srcpkgs/s6-networking/template

```diff
@@ -1,20 +1,28 @@
 # Template file for 's6-networking'
 pkgname=s6-networking
 version=2.3.1.2
-revision=2
+revision=3
 build_style=configure
-configure_args="--libdir=/usr/lib --enable-ssl=libressl
+configure_args="--libdir=/usr/lib $(vopt_if libressl --enable-ssl=libressl)
+ $(vopt_if bearssl --enable-ssl=bearssl)
  --with-sysdeps=${XBPS_CROSS_BASE}/usr/lib/skalibs/sysdeps
  --with-lib=${XBPS_CROSS_BASE}/usr/lib --bindir=/usr/bin"
-makedepends="execline-devel libressl-devel skalibs-devel s6-devel s6-dns-devel"
+makedepends="execline-devel $(vopt_if libressl libressl-devel) skalibs-devel
+ s6-devel s6-dns-devel $(vopt_if bearssl bearssl-devel)"
 short_desc="Suite of small network utilities for Unix systems"
 maintainer="lemmi <lemmi@nerd2nerd.org>"
 license="ISC"
-homepage="https://skarnet.org/software/${pkgname}/"
-changelog="https://skarnet.org/software/s6-networking/upgrade.html"
-distfiles="https://skarnet.org/software/${pkgname}/${pkgname}-${version}.tar.gz"
+homepage="https://skarnet.org/software/s6-networking"
+changelog="${homepage}/upgrade.html"
```

Comment:
I don't like that, now we can't copy/paste the changelog when looking at the template.

Reply:
Fixed.